docs-source/content/quickstart/install.md

```diff
@@ -37,40 +52,77 @@ $ helm install stable/traefik \
   --wait
 ```
 
+
+For helm version 3.x, you should have added the stable repository if it has not been done already.
+
+```bash
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+```
```

Comment:
Helm is also used to install the operator, should the same instructions be included there?